src/ac_power_flow_jacobian.jl

```diff
@@ -198,6 +198,82 @@ function _create_jacobian_matrix_structure_bus!(rows::Vector{Int32},
     return nothing
 end
 
+function _create_jacobian_matrix_structure_lcc(data::ACPowerFlowData, rows::Vector{Int32},
+    columns::Vector{Int32},
+    values::Vector{Float64},
+    num_buses::Int)
+    for (i, (fb, tb)) in enumerate(zip(data.lcc.rectifier_bus, data.lcc.inverter_bus))
+        push!(rows, 2 * fb - 1)
+        push!(columns, 2 * fb - 1)
+        push!(values, 0.0)
+
+        push!(rows, 2 * fb)
+        push!(columns, 2 * fb - 1)
+        push!(values, 0.0)
+
+        push!(rows, 2 * fb - 1)
+        push!(columns, num_buses * 2 + (i - 1) * 4 + 1)
+        push!(values, 0.0)
+
+        push!(rows, 2 * fb - 1)
+        push!(columns, num_buses * 2 + (i - 1) * 4 + 3)
+        push!(values, 0.0)
+
+        push!(rows, 2 * fb)
+        push!(columns, num_buses * 2 + (i - 1) * 4 + 1)
+        push!(values, 0.0)
+
+        push!(rows, 2 * fb)
+        push!(columns, num_buses * 2 + (i - 1) * 4 + 3)
+        push!(values, 0.0)
+
+        push!(rows, num_buses * 2 + (i - 1) * 4 + 1)
+        push!(columns, 2 * fb - 1)
+        push!(values, 0.0)
+
+        push!(rows, num_buses * 2 + (i - 1) * 4 + 2)
+        push!(columns, 2 * fb - 1)
+        push!(values, 0.0)
+
+        push!(rows, num_buses * 2 + (i - 1) * 4 + 2)
+        push!(columns, 2 * tb - 1)
+        push!(values, 0.0)
+
+        push!(rows, num_buses * 2 + (i - 1) * 4 + 1)
+        push!(columns, num_buses * 2 + (i - 1) * 4 + 1)
+        push!(values, 0.0)
+
+        push!(rows, num_buses * 2 + (i - 1) * 4 + 1)
+        push!(columns, num_buses * 2 + (i - 1) * 4 + 3)
+        push!(values, 0.0)
+
+        push!(rows, num_buses * 2 + (i - 1) * 4 + 2)
+        push!(columns, num_buses * 2 + (i - 1) * 4 + 1)
+        push!(values, 0.0)
+
+        push!(rows, num_buses * 2 + (i - 1) * 4 + 2)
+        push!(columns, num_buses * 2 + (i - 1) * 4 + 2)
+        push!(values, 0.0)
+
+        push!(rows, num_buses * 2 + (i - 1) * 4 + 2)
+        push!(columns, num_buses * 2 + (i - 1) * 4 + 3)
+        push!(values, 0.0)
+
+        push!(rows, num_buses * 2 + (i - 1) * 4 + 2)
+        push!(columns, num_buses * 2 + (i - 1) * 4 + 4)
+        push!(values, 0.0)
+
+        push!(rows, num_buses * 2 + (i - 1) * 4 + 3)
+        push!(columns, num_buses * 2 + (i - 1) * 4 + 3)
+        push!(values, 1.0)
+
+        push!(rows, num_buses * 2 + (i - 1) * 4 + 4)
+        push!(columns, num_buses * 2 + (i - 1) * 4 + 4)
+        push!(values, 1.0)
```

Comment:
Is it possible to define a helper function to streamline some of this implementation? It all feels a little magical for me, maybe some comments would also be useful